robot_modeling/DQ_SerialManipulator.m

```diff
@@ -109,34 +131,6 @@ function set_effector(obj,effector)
             obj.effector = DQ(effector);
         end
         
-        function x = raw_fkm(obj,q, ith)
```

Comment:
This method does not depend on any convention. Therefore, it should be maintained as an abstract method to enforce that all subclasses implement it.

Reply:
Hi @bvadorno, (assuming you are talking about raw_fkm())

the methods raw_fkm(), raw_pose_jacobian(), and raw_pose_jacobian_derivative() are abstracs methods in `DQ_SerialManipulator` and all of them are implemented in `DQ_SerialManipulatorDH`.

Reply:
Hi @juanjqo,

Yes, I was referring to `raw_fkm()`. My comment was because I was under the impression that it had been removed from the `DQ_SerialManipulator`. I guess I missed it. I'm going to resolve this conversation, then!

Thanks,
Bruno